AppBuilder/platform/views/viewProperties/ABViewPropertyFilterData.js

```diff
@@ -1009,44 +1026,41 @@ export default class ABViewPropertyFilterData extends ABViewProperty {
       };
 
       let init = (options) => {
-         this.filter_popup = webix.ui({
-            view: "popup",
-            id: ids.component,
-            width: 800,
-            hidden: true,
-            body: this.rowFilter.ui,
-         });
+         // this.filter_popup = webix.ui({
+         //    view: "popup",
+         //    id: ids.component,
+         //    width: 800,
+         //    hidden: true,
+         //    body: this.rowFilter.ui,
+         // });
 
          // register callbacks:
          for (var c in logic.callbacks) {
```

Comment:
do we still support callbacks?

Reply:
There are .change and .save events like EventEmitter.

I don't sure do we still need to callback style ?